packages/flutter/test/material/navigation_bar_test.dart

```diff
@@ -0,0 +1,429 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  setUp(() async {
+    WidgetsBinding.instance!.focusManager.highlightStrategy =
```

Comment:
Please include a comment that explains why we need to do this

Reply:
Not needed, removing